test/system/trader.js

```diff
@@ -297,7 +300,7 @@ class Trader {
         },
         error => {
           if (options.expectFail) {
-            console.log('Create Order failed but this was expected.');
+            console.log('Create Order failed due to', options.expectFailReason);
```

Comment:
The main purpose of options.expectFailReason is to set the Error message (line 290 above) for when the createOrder is expected to fail, but doesn't.
This will make it easier to determine what, if anything, causes this scenario to fail